pkg/config/setup/config.go

```diff
@@ -1312,10 +828,568 @@ func InitConfig(config pkgconfigmodel.Config) {
+
+	// If set, the agent will look in this path for docker container log files.  Use this option if
+	// docker's `data-root` has been set to a custom path and you wish to ingest docker logs from files. In
+	// order to check your docker data-root directory, run the command `docker info -f '{{.DockerRootDir}}'`
+	// See more documentation here:
+	// https://docs.docker.com/engine/reference/commandline/dockerd/.
+	config.BindEnvAndSetDefault("logs_config.docker_path_override", "")
+
+	config.BindEnvAndSetDefault("logs_config.auditor_ttl", DefaultAuditorTTL) // in hours
+	// Timeout in milliseonds used when performing agreggation operations,
+	// including multi-line log processing rules and chunked line reaggregation.
+	// It may be useful to increase it when logs writing is slowed down, that
+	// could happen while serializing large objects on log lines.
+	config.BindEnvAndSetDefault("logs_config.aggregation_timeout", 1000)
+	// Time in seconds
+	config.BindEnvAndSetDefault("logs_config.file_scan_period", 10.0)
+
+	// Controls how wildcard file log source are prioritized when there are more files
+	// that match wildcard log configurations than the `logs_config.open_files_limit`
+	//
+	// Choices are 'by_name' and 'by_modification_time'. See config_template.yaml for full details.
+	//
+	// WARNING: 'by_modification_time' is less performant than 'by_name' and will trigger
+	// more disk I/O at the wildcard log paths
+	config.BindEnvAndSetDefault("logs_config.file_wildcard_selection_mode", "by_name")
+}
+
+func vector(config pkgconfigmodel.Config) {
+	// Vector integration
+	bindVectorOptions(config, Metrics)
+	bindVectorOptions(config, Logs)
+}
+
+func cloudfoundry(config pkgconfigmodel.Config) {
+	// Cloud Foundry
+	config.BindEnvAndSetDefault("cloud_foundry", false)
+	config.BindEnvAndSetDefault("bosh_id", "")
+	config.BindEnvAndSetDefault("cf_os_hostname_aliasing", false)
+	config.BindEnvAndSetDefault("cloud_foundry_buildpack", false)
+}
+
+func containerd(config pkgconfigmodel.Config) {
+	// Containerd
+	config.BindEnvAndSetDefault("containerd_namespace", []string{})
+	config.BindEnvAndSetDefault("containerd_namespaces", []string{}) // alias for containerd_namespace
+	config.BindEnvAndSetDefault("containerd_exclude_namespaces", []string{"moby"})
+	config.BindEnvAndSetDefault("container_env_as_tags", map[string]string{})
+	config.BindEnvAndSetDefault("container_labels_as_tags", map[string]string{})
+}
+
+func cri(config pkgconfigmodel.Config) {
+	// CRI
+	config.BindEnvAndSetDefault("cri_socket_path", "")              // empty is disabled
+	config.BindEnvAndSetDefault("cri_connection_timeout", int64(1)) // in seconds
+	config.BindEnvAndSetDefault("cri_query_timeout", int64(5))      // in seconds
+}
+
+func kubernetes(config pkgconfigmodel.Config) {
+	// Kubernetes
+	config.BindEnvAndSetDefault("kubernetes_kubelet_host", "")
+	config.BindEnvAndSetDefault("kubernetes_kubelet_nodename", "")
```

Comment:
This config value is reachable, but in a method called `GetEKSFargateNodename`. I think it's safe to assume we'll never reach that code since lambda is not eks fargate.